implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -261,6 +265,8 @@ pub enum OckamSubcommand {
     Space(SpaceCommand),
     Project(ProjectCommand),
     Admin(AdminCommand),
+    #[cfg(feature = "orchestrator")]
```

Comment:
I am not if it is really correct to add the `Orchestrator` feature. We haven't been using anything like this for the `Space` and `Project` command and yet they also talk to the `Orchestrator`.

Reply:
From my POV that's fine, as this is a tacit proposal that we should start labeling functionality that are quite specific to using our SAAS backend from things that can work without it - not least of which because that's useful information to convey in our documentation and help output. Even if the default binary includes the whole kitchen sink, this practice would provide us better indications when reading or reviewing code and also force us to provide better seams in the modules and implementation code. At least, it would as long as we periodically use `cargo check --no-default-features`, which currently fails in a few spots even prior to my proposal here.

If you want me to pull that proposal out to a separate conversation that's fine, but it's far easier for me to design net-new code with that practice in mind and strip it out if we reach a team decision to not pursue it, than to retrofit it later.